spec/presenters/fully_signed_in_modal_presenter_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+describe FullySignedInModalPresenter do
+  subject(:presenter) { FullySignedInModalPresenter.new(10) }
+
+  it 'implements SessionTimeoutWarningModalPresenter' do
+    SessionTimeoutWarningModalPresenter.instance_methods(false).each do |method|
+      expect(presenter.send(method)).to be
+    end
+  end
+end
```

Comment:
there is no newline at end of file -- update sublime? :) https://forum.sublimetext.com/t/make-saving-newline-at-eof-the-installation-default/9842